### PR TITLE
k/examples repo automation added

### PR DESCRIPTION
**Pull request by idvoretskyi**
PTAL @ixdy 

Fixes https://github.com/kubernetes/examples/issues/218
cc @ahmetb

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -206,6 +207,9 @@ plugins:
 
   kubernetes/dns:
   - trigger
+  
+  kubernetes/examples:
+  - trigger
```
Comment (ahmetb) [inline prow/plugins.yaml]: we might need `approve` here too. I've been trying to get OWNERS files for each sample.

Comment (idvoretskyi) [reply prow/plugins.yaml]: @ahmetb fixed https://github.com/kubernetes/test-infra/pull/7455/commits/8f9ac437ef7d5dbc7b7e3cdc7ac0b272e6b91924

Comment (ixdy) [reply prow/plugins.yaml]: I think you only need `trigger` if you are running tests on prow, which doesn't seem to be the case currently.

**Pull request merged**